Carmunication/Models/DummyData/Point2.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Point2.swift
+//  Carmunication
+//
+//  Created by 김동현 on 2023/10/16.
+//
+
+import Foundation
+
+struct Point2 {
+    var pointId: Int?
```

Comment:
Swift에서는 약어의 경우 보통 `pointID`와 같이 사용하는 것 같아요
아니면 PointIdentifier와 같이 풀어서 작성하기도 하구요

```suggestion
    var pointID: Int?
```

Reply:
네네 바꿔놓겠습니다.